pdns/recursordist/docs/appendices/EOL.rst

```diff
@@ -1,5 +1,10 @@
 End of life statements
-======================
+=====================
```

Comment:
did this remove one actually required =?

Reply:
yes, tx